ContentProtection/OfflinePlayReadyAndWidevine/src/main/java/sample/OfflinePlayReadyAndWidevine.java

```diff
@@ -14,56 +14,29 @@
 import java.util.concurrent.Callable;
 import java.util.Arrays;
 
+import com.azure.core.management.exception.ManagementException;
```

Comment:
We should make sure to explain in the Readme.md why this core library is required and why we have used it.